install.sh

```diff
@@ -260,8 +260,8 @@ else
 fi
 
 # scylla-conf
-install -d -m755 "$retc"/scylla
-install -d -m755 "$retc"/scylla.d
+install -Z -d -m755 "$retc"/scylla
+install -Z -d -m755 "$retc"/scylla.d
```

Comment:
How about

```sh
install() {
    command install -Z "$@"
}
```

So we only need to change one place, and easy to backport.

Reply:
@avikivity pushed new version which defines install() function.

Reply:
Ah, I found a bug of current PR, current script does not able to install files with correct security context (at least on CentOS7).
Currently "$retc" is extracted to "//etc", since $root="/" and $retc="$root/etc", seems like this does not work for install -Z.

This command worked finely:
`install -Z -m644 dist/common/systemd/scylla-server.service.d/dependencies.conf -Dt /etc/systemd/system/scylla-server.service.d/`
but this didn't work:
`install -Z -m644 dist/common/systemd/scylla-server.service.d/dependencies.conf -Dt //etc/systemd/system/scylla-server.service.d/`

I think we need to normalize the path, probably by using `realpath`.

Reply:
Puthed new version which normalized install path.